docs/user-defined-ls/vscode-css-language-server.md

```diff
@@ -1,10 +1,19 @@
 # CSS Language Server
 
-You can use [Visual Studio Code's CSS language server](https://github.com/microsoft/vscode-css-languageservice) by following these instructions:
+You can use the [Visual Studio Code's CSS language server](https://github.com/microsoft/vscode-css-languageservice) to benefit with `CSS`, `LESS` and `SCSS` support:

```

Comment:
You can leverage the [Visual Studio Code CSS Language Server](https://github.com/microsoft/vscode-css-languageservice) to enhance your development experience with support for `CSS`, `LESS`, and `SCSS`.